pkg/serverless/metrics/enhanced_metrics.go

```diff
@@ -33,9 +31,8 @@ func getOutOfMemorySubstrings() []string {
 	}
 }
 
-// generateEnhancedMetricsFromRegularLog generates enhanced metrics from a LogTypeFunction message
-func generateEnhancedMetricsFromFunctionLog(message aws.LogMessage, tags []string, metricsChan chan []metrics.MetricSample) {
-	logString := message.StringRecord
+// GenerateEnhancedMetricsFromFunctionLog generates enhanced metrics from a LogTypeFunction message
```

Comment:
Does this need to be public?

Reply:
Yes I think so, since this method is called from the new logs package. (but every other methods are now private)